Wellthier/Cells/PostCell.h

```diff
@@ -16,6 +16,7 @@ NS_ASSUME_NONNULL_BEGIN
 
 @property (weak, nonatomic) IBOutlet UIButton *toProfileHiddenButton;
 @property (weak, nonatomic) IBOutlet UILabel *displayName;
+@property (weak, nonatomic) IBOutlet UILabel *username;
```

Comment:
nit: for those UILabel items, you should name them with Label as postfix. For example, usernameLabel.
Same for UITexView, it should be postTextView.